src/main/ApplicationImpl.cpp

```diff
@@ -399,20 +399,26 @@ ApplicationImpl::start()
         throw std::invalid_argument("NODE_IS_VALIDATOR not set");
     }
 
-    if (mConfig.METADATA_OUTPUT_STREAM != "" && mConfig.NODE_IS_VALIDATOR)
+    if (mConfig.METADATA_OUTPUT_STREAM != "" && mConfig.isNetworkedValidator())
```

Comment:
it seems that this is the only place in the entire code base that we use `isNetworkedValidator`, I think that adding a variable with that name here (and using it for both conditions) avoids introducing a new concept to the code base

Reply:
Alright, I've done that.